app/templates/events/view.hbs

```diff
@@ -1,6 +1,74 @@
 {{#unless (includes session.currentRouteName 'events.view.edit')}}
-  {{!-- Tabs for the event view page will come here --}}
-  {{outlet}}
+  <div class="ui grid stackable">
+    <div class="row">
+      <div class="four wide column">
+        <h2 class="ui header">Fossasia Test</h2>
+      </div>
+      <div class="twelve wide column {{unless device.isMobile 'right aligned'}}">
+        {{#if device.isMobile}}
+          <button class="ui button"><i class="unhide icon"></i></button>
+          <div class="ui icon buttons">
+            <button class="ui button "><i class="check icon"></i></button>
+            <button class="ui button "><i class="copy icon"></i></button>
+          </div>
+          <button class="ui red button icon"><i class="trash icon"></i></button>
+        {{else}}
+            <button class="ui button labeled icon small">
+              <i class="unhide icon"></i>
+              {{t 'Preview'}}
+            </button>
+            <div class="ui small labeled icon buttons">
+              <button class="ui button ">
+                <i class="check icon"></i>
+                {{t 'Publish'}}
+              </button>
+              <button class="ui button ">
+                <i class="copy icon"></i>
+                {{t 'Copy'}}
+              </button>
+            </div>
+            <button class="ui red button labeled icon small">
+              <i class="trash icon"></i>
+              {{t 'Delete'}}
+            </button>
+        {{/if}}
+      </div>
+    </div>
+  </div>
+
+  <div class="row" style="padding-top: 1em">
+    <div class="sixteen wide column">
+      {{#if device.isMobile}}
+        {{#ui-dropdown class='vertical selection'}}
+          <i class="dropdown icon"></i>
+          <div class="default text">{{t 'Overview'}}</div>
```

Comment:
@utkarshnath don't make things too complex than they have to be. In this implementation based on the route you need to manually get the title and set what is being displayed etc. And there's a lot of code duplication too. Please remove this implementation and make a simple one as shown below.